tools/logger/logger.c

```diff
@@ -141,6 +143,7 @@ static int configure_uart(const char *file, unsigned int baud)
 
 int main(int argc, char *argv[])
 {
+	static const char opt_frm[] = "ho:i:l:ps:c:u:tev:rdLf:F:g:";
 	struct convert_config config;
```

Comment:
Lets name opt_frm -> optstring as the man page of getopt(3) names it.

Reply:
changed